doc/user-guide/groups_management.md

```diff
@@ -0,0 +1,75 @@
+## Groups Management
+
+IAM provides a group management system. 
+Only users with ``ROLE_ADMIN`` authority can create new groups, remove existing groups
```

Comment:
Only administrator users can create new groups, remove existing groups and change the group membership.